furbysource.asm

```diff
@@ -506,4 +506,285 @@ InitIO:
       STA   Mili_sec    ;also preset IRQ timer
 
         CLI             ;Enable IRQ
-A-20
\ No newline at end of file
+A-20
+      JSR   Kick_IRQ    ;wait for interrupt to restart
+      JSR   TI_reset    ;go init TI (uses 'Cycle_timer')
+
+; Preset motor speed, assuming mid battery life, we eat the pulse width
+; so that the motor wont be runing at 6 volts and burn out. We then
+; predict what the pulse width should be for any voltage.
+
+;     LDA   #Mpulse_on  ;preset motor speed
+      LDA   #ll
+      STA   Mon_len           ;set motor on pulse timing
+
+      LDA   #05         ;
+      STA   Moff_len    ;
+
+;ÈÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍ_____ÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍ
+;* 'Diagnostics and celibration Routine                   *
+;ÈÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍ_ÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍÍ
+;
+
+      Include           Dieg7.asm   ;asm file
+
+; ****** Only called by diagnostic speech routines *********
```

Comment:
There definitely some oddities in the source file on lines 525-531 - lots of "a" characters that look like they're being rendered as something almost indistinguishable from "e" characters.  (I don't think there are faults in your transcription here, I just want to make a note for future reference that there may be some trouble on these pages.)

FIXME: check back here when more of the PDF is transcribed to see if a `Dieg7.asm` file exists or whether it should be `Diag7.asm` instead

Reply:
A-116 - A-125 is the source code of that file, and it seems like that has the same issue. Looks like an `a` on most pages.

Reply:
Did notice this too, but if you copy/past from the PDF, it copies as 'a's, so I went with that